src/main/flight/mixer.c

```diff
@@ -607,9 +621,25 @@ FAST_CODE_NOINLINE void mixTable(timeUs_t currentTimeUs)
     // Find min and max throttle based on conditions. Throttle has to be known before mixing
     calculateThrottleAndCurrentMotorEndpoints(currentTimeUs);
 
+    static bool turtleModeStarted = false;
     if (isFlipOverAfterCrashActive()) {
-        applyFlipOverAfterCrashModeToMotors();
+        static float tiltAngleAtStart = 1.0f;
+        float flipAngleModifier = 1.0f;
+#ifdef USE_ACC
+        const float tiltAngle = getCosTiltAngle(); // -1 if inverted, 0 when 90 degrees, 1 when flat and upright
+        if (!turtleModeStarted) {
+            tiltAngleAtStart = tiltAngle;
+        }
+        turtleModeStarted = true;
+        // send a factor that attenuates motor drive to zero as the flip approaches 90 degrees of rotation
+        // automatically stops the motors, even though momentum typically causes more rotation than intended
+        flipAngleModifier = fmaxf(1.0f - fabsf(tiltAngleAtStart - tiltAngle), 0.0f);
```

Comment:
You are using cos(angle) - cos(angle2), not angle difference. Threshold angle will be different, depending on starting position. But this approximation may be fine in practise

Reply:
Yes it is a crude approximation :-)
I tested starting at 135 degrees (inverted at 45 degrees pitch away from horizon). The quad turned and auto-stop appeared too work.  It will do :-)